Exception/AckException.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Seems odd to have an Exception for Acknowledging that the message was processed successfully.  I suppose there are situations when you'd want it but it seems odd...